pyproject.toml

```diff
@@ -30,15 +30,3 @@ Homepage = "https://github.com/google/gcp_scanner"
 
 [project.scripts]
 gcp-scanner = "gcp_scanner.scanner:main"
-
-[tool.hatch.build]
-exclude = [
```

Comment:
Could you please add these files too into the exclusion list in the MANIFEST.in

Reply:
I didn't add them because setuptools includes only .py files
But we can add them to be sure

Reply:
I see, anyway thanks for doing that.